spec.html

```diff
@@ -30021,14 +30020,10 @@ <h1>Properties of the Function Constructor</h1>
       <ul>
         <li>is itself a built-in function object.</li>
         <li>has a [[Prototype]] internal slot whose value is %Function.prototype%.</li>
+        <li oldids="sec-function.length">has a *"length"* property whose initial value is *1*<sub>𝔽</sub>.</li>
```

Comment:
```suggestion
        <li oldids="sec-function.length">has a *"length"* property whose value is *1*<sub>𝔽</sub>.</li>
```
it doesn't seem like we say "initial" most places, since that's implied by it being configurable?

Reply:
I've dropped the word "initial" from spots where the corresponding source text doesn't say "initial".

Do you mean "most places where we declare the value of a property", or "most places where we declare the value of the `"length"` property"?

My preference (for all properties) would be to say "initial" if and only if the value of the property could change (i.e. `[[Writable]]` and/or `[[Configurable]]` is `*true*`), but that's a different PR.

Reply:
I haven’t audited but I’d expect we rarely use it at all. If we did your preference we’d be adding it almost everywhere - I’d prefer it be implied by the property descriptor.

Reply:
I count 232 spots where the status quo spec declares the [initial] value of a property: 157 say "initial" and 75 don't.

Reply:
Sure - but what's the point of saying it at all? Almost every user-visible thing in the spec is "initial".

Reply:
> Sure - but what's the point of saying it at all?

Presumably the point was to remind/alert the reader that the value might be different later.

> Almost every user-visible thing in the spec is "initial".

Indeed. But I don't think the spec does a good job of conveying that.

[18 ECMAScript Standard Built-in Objects](https://tc39.es/ecma262/#sec-ecmascript-standard-built-in-objects) should probably have a paragraph saying that the statements in sections 18-28 describe only the *initial* state of a realm (i.e., the state it has when InitializeHostDefinedRealm() returns), but that subsequent processing can render many of those statements false.  And then a Note with some examples. If we had that, then I wouldn't much mind deleting all the "initial"s as you'd prefer.